macq/extract/esam.py

```diff
@@ -128,12 +141,12 @@ def add_not_iseff(post_state: State):
                         if fluent in post_state.fluents.keys():
                             if not post_state.fluents[fluent]:
                                 not_eff = Var(literals2index[flu], False)
-                                cnf_ef_as_set[a.name].add(Or([not_eff]))
+                                # cnf_ef_as_set[a.name].add(Or([not_eff]))  TODO change this
```

Comment:
Remove TODOs from submitted code (if we want to push this to the main branch we need to remove these code parts).